demo/cli/src/lib.rs

```diff
@@ -47,20 +47,23 @@ extern crate log;
 
 pub mod error;
 
-use std::sync::Arc;
 use demo_primitives::Hash;
-use demo_runtime::{Block, BlockId, UncheckedExtrinsic, GenesisConfig,
-	ConsensusConfig, CouncilConfig, DemocracyConfig, SessionConfig, StakingConfig,
-	TimestampConfig};
+use demo_runtime::{
```

Comment:
It is only possible to configure rustfmt (at the moment) to either have importans in the first line and then match up the second line to it (`imports_indent = "Visual"`) or make one-tabbed blocks like this (`imports_indent = "Block"`)